src/util.ts

```diff
@@ -70,11 +70,13 @@ export class Util {
     /**
      * Given a pkg path of any kind, transform it to a roku-specific pkg path (i.e. "pkg:/some/path.brs")
      */
-    public getRokuPkgPath(pkgPath: string) {
+    public sanitizePkgPath(pkgPath: string) {
         pkgPath = pkgPath.replace(/\\/g, '/');
-        return 'pkg:/' + pkgPath;
+        if (!pkgPath.startsWith('pkg:/')) {
```

Comment:
Maybe add `libpkg:`

Reply:
Handled in 50a155549b9b4a9fbacbef8ecc769109d6abc66e